Lavinia-api/Data/NOInitializer.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using LaviniaApi.Models;
+using LaviniaApi.Utilities;
+using Microsoft.Extensions.Logging;
+
+namespace LaviniaApi.Data
+{
+    // API v2
+    public static class NOInitializer
```

Comment:
This should be refactored to use more methods for readability (and potentially re-usability in other contexts). Essentially every place you write a comment, you should have a function that does what the comment describes, preferably in such a way that it doesn't matter what context the initializer is dealing with.